chart/ViewController.swift

```diff
@@ -10,124 +10,90 @@ import UIKit
 
 class ViewController: UIViewController {
     
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         
         self.view.backgroundColor = UIColor(named: "ColorBackground")
         
-        shape()
-        shape2()
-        shape3()
+        // Set Percentage
+        shape(totalPercentage: 100, array: [(UIColor(named: "ColorA"), 10), (UIColor(named: "ColorB"), 20), (UIColor(named: "ColorC"), 50), (UIColor.clear, 20)])
```

Comment:
the remove the hard coded part of (UIcolor.clear, 20). since the percentages are the only input and are supposed to be dynamic, the buffer value met change.